classes/Application.php

```diff
@@ -92,6 +92,10 @@ public function __construct($basePath, Environment $environment)
         $this->register(new ApplicationServiceProvider);
 
         $this->registerGlobalErrorHandler($this);
+
+        if($timezone = $this->config('application.date_timezone')) {
```

Comment:
@davethegr8 

The build is failing because of a missing space after `if`, see https://travis-ci.org/opencfp/opencfp/jobs/112294546#L591.